### DISCUSSION TITLE
Resource Monitor - Horizontal

### DISCUSSION THREAD

**Discussion by GreyHound55** [crystian/ComfyUI-Crystools#119] [Ideas]
With the overhauled UI and most items are now in the top-bar, can the Resource Monitor be designed with a landscape (horizontal) option ? 

It currently is in Portrait (vertical) mode, which increaes the height of the top-bar, thus reducing the footprint for workflows. 

Even better, if it was Horizontal and could be positioned in the center-section of the top-bar (which appears to be unused) ... this would be ideal ! 

Thx!

**Comment by crystian** [comment]
WIP ;)

**Reply by GreyHound55** [reply]
Thanks, that would be fantastic !
